src/wled2.html

```diff
@@ -29,6 +40,49 @@
     oneditprepare: function() {
       $("#node-input-address").change({ currentEffect: this.effect, currentPalette: this.palette }, loadDropdowns);
 
+      // When a label is clicked, toggle it's enabled/disabled status, if possible
+      $("label").click(event => {
+        // Get the element this click event applies to
+        const label = $(event.currentTarget);
+        const stateCheckbox = label.find("input").first();
+        // If there isn't a checkbox inside the label, skip
+        if (stateCheckbox.length !== 0) {          
+          const elementFromLabel = $(`#${label.attr("for")}`);
+          // If there isn't a field attached to the label, skip
+          if (elementFromLabel.length == 1) {
+            // Get the element that was actually clicked for this event, not it's parent.
+            const clickedElement = $(event.target);
+            // Flip state, checked == true, which means the field is enabled, so disabled is false.
+            const newState = !stateCheckbox.prop("checked");
+            // Check if what we clicked was the checkbox
+            if (clickedElement.attr("id") !== stateCheckbox.attr("id")) {
+              // If we didn't click the checkbox, flip the state of the checkbox, and the disabled state of the input
+              stateCheckbox.prop("checked", newState);
+              // Double flipped because the value we got before was when we hadn't flipped the checkbox.
+              clickedElement.prop("disabled", !newState);
+            } else {
+              // If we did click the checkbox, just update the disabled state of the input
+              clickedElement.prop("disabled", newState);
+            }
+          }
+        }
+      });

```

Comment:
This is an awful lot of code complexity to make clicking on the label toggle the checkbox.

At the moment the labels are assigned as "for" the actual data control, but in practice this is not particularly useful. Rather than introduce all of the above, instead either:

1. Change the label's "for" attribute to attach to the enable checkbox and get the label click behaviour for free
2. Don't bother trying to make clicking on the label toggle the checkbox

The only downside to approach #1 above will be possible confusion when used with a screen reader, since it will read the label as being attached to the checkbox not the primary control. In practice I think that's an ok tradeoff to remove all the above complexity.